MacOS login credentials not saved

f1viewer will not safe login credentials in de MacOS keychain. 
After selecting save it'll log in, and then give a 'No keyring configured' message.
I'm running MacOS catalina on both machines I'm having this issue on.

@SoMuchForSubtlety I'm also experiencing the same problem on Catalina.
installed via Homebrew
```
$ f1viewer --version
Version:     1.1.0
Git commit:  b6963937df1c3608c3bbf0cbc09cf28123893564
Built:       2020-06-29T15:29:14Z
```
When I start up the app I see the following entries in the log
```
2020/07/04 15:25:39 [ERROR] Could not access credential store: Specified keyring backend not available
2020/07/04 15:25:39 [ERROR] No keyring configured
```

 I've done a little digging and I found it works as expected if I run the master branch with `go run ./...` then I get prompted for permission to access the keychain as I would expect. I never see any prompting running the released version.
<img width="449" alt="f1viewer" src="https://user-images.githubusercontent.com/147524/86520479-6d3ebd00-be0a-11ea-8062-cf519c8315f8.png">

I'm not really sure where else to take this debugging but if I can be of any help in testing or collecting info for you please let me know.

I downloaded the binary v1.1.0,  (Version:     1.1.0
Git commit:  b6963937df1c3608c3bbf0cbc09cf28123893564
Built:       2020-06-29T15:29:14Z
)
and I also get the same error :

> 2020/07/08 10:24:09 [ERROR] Could not access credential store: Specified keyring backend not available
> 2020/07/08 10:24:09 [ERROR] No keyring configured

OS: MacOS 10.15.5

Whether in user or sudo mode, I couldn't get the permission pop-up either.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -4,28 +4,32 @@ jobs:
   test:
     strategy:
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
-    runs-on: ${{ matrix.platform }}
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
     steps:
-    - name: Install Go
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.14.x
-    - name: Checkout code
-      uses: actions/checkout@v1
-    - name: Test
-      run: go test ./...
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.x
+      - name: Checkout code
+        uses: actions/checkout@v1
+      - name: Test
+        run: go test ./...
   build:
     strategy:
       matrix:
-        platform: [ubuntu-latest, macos-latest, windows-latest]
-    runs-on: ${{ matrix.platform }}
+        os: [ubuntu-latest, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
     steps:
-    - name: Install Go
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.14.x
-    - name: Checkout code
-      uses: actions/checkout@v1
-    - name: Build
-      run: go build -v .
+      - name: Install Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.x
+      - name: Checkout code
+        uses: actions/checkout@v1
+      - name: Build
+        run: go build -v .
+      - uses: actions/upload-artifact@v2
+        with:
+          name: ${{ matrix.os }}-f1viewer
+          path: f1viewer*

--- a/.github/workflows/release.yml
+++ b/.github/workflows/release.yml
@@ -3,28 +3,43 @@ name: release
 on:
   push:
     tags:
-      - '*'
+      - "*"
 
 jobs:
   goreleaser:
     runs-on: ubuntu-latest
     steps:
-      -
-        name: Checkout
+      - name: Checkout
         uses: actions/checkout@v2
-      -
-        name: Unshallow
+      - name: Unshallow
         run: git fetch --prune --unshallow
-      -
-        name: Set up Go
+      - name: Set up Go
         uses: actions/setup-go@v1
         with:
           go-version: 1.14.x
-      -
-        name: Run GoReleaser
+      - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v1
         with:
           version: latest
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN_GORELEASER }}
+  macOs-build:
+    runs-on: macos-latest
+    steps:
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\/v/}
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.x
+      - name: Build
+        run: ./macOS-build.sh
+      - name: Upload artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: f1viewer_${{ steps.get_version.outputs.VERSION }}_macOS
+          path: f1viewer_${{ steps.get_version.outputs.VERSION }}_macOS_64-bit.tar.gz

--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -22,13 +22,7 @@ checksum:
 snapshot:
   name_template: "{{ .Tag }}-next"
 changelog:
-  sort: asc
-  filters:
-    exclude:
-    - '^docs:'
-    - '^test:'
-    - Merge pull request
-    - Merge branch
+  skip: true
 nfpms:
   -
     description: TUI client for F1TV
@@ -44,16 +38,17 @@ nfpms:
       - mpv
     recommends:
       - xclip
-brews:
-  -
-    description: TUI client for F1TV
-    homepage: https://github.com/SoMuchForSubtlety/f1viewer/
-    folder: Formula
-    github:
-      owner: SoMuchForSubtlety
-      name: homebrew-tap
-    commit_author:
-      name: goreleaserbot
-      email: goreleaser@carlosbecker.com
-    dependencies:
-      - mpv
+# does not work due to cgo not being supported by goreleaser
+# brews:
+#   -
+#     description: TUI client for F1TV
+#     homepage: https://github.com/SoMuchForSubtlety/f1viewer/
+#     folder: Formula
+#     github:
+#       owner: SoMuchForSubtlety
+#       name: homebrew-tap
+#     commit_author:
+#       name: goreleaserbot
+#       email: goreleaser@carlosbecker.com
+#     dependencies:
+#       - mpv

--- a/macOS-build.sh
+++ b/macOS-build.sh
@@ -1,0 +1,10 @@
+#!/bin/bash
+now=$(date --iso-8601=seconds)
+go build -v -ldflags="-X main.version=${GITHUB_REF/refs\/tags\/v/} -X main.commit=${GITHUB_SHA} -X main.date=$now"
+if [ $? -eq 0 ]; then
+    echo build OK, creating archive
+    tar -zcvf f1viewer_"${GITHUB_REF/refs\/tags\/v/}"_macOS_64-bit.tar.gz LICENSE README.md f1viewer
+else
+    echo FAILED TO BUILD
+    exit 1
+fi

